modules/sandboxed-containers-deleting-operator-deployment-cr.adoc

```diff
@@ -0,0 +1,38 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-operator-deployment-cr_{context}"]
+= Deleting the {sandboxed-containers-operator} deployment custom resource
+
+To uninstall {sandboxed-containers-first}, you must first delete the {sandboxed-containers-first} custom resource `KataConfig`. This removes and uninstalls the `kata` runtime and its related resources from your cluster.
+
+.Prerequisites
+
+* You have {product-title} 1.0.0 installed.
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. From the {product-title} web console, select `openshift-sandboxed-containers` from the *Projects* list.
+. Navigate to the *Operators* → *Installed Operators* page.
+. Click *{sandboxed-containers-first}*.
+. Click the *{sandboxed-containers-operator}* tab.
+. Click the scroll-down list in the *Operator Details*, and click *Delete `KataConfig`*.
+. Click *Delete* in the confirmation window.
+. Delete all `pod`, `deployment`, `DeploymentConfig`, `ReplicaSet`, `ReplicationController`, and `StatefulSet` objects that use `kata` as `runtimeClass`.
+.. Use the OpenShift CLI (oc) to retrieve all pods that use `kata` as `runtimeClassName` by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -o json | jq -r '.items[] | select(.spec.runtimeClassName | test("kata")).metadata.name'
```

Comment:
This requires `jq`, which is not listed in the prerequisites ;-)

Reply:
As a part of the prerequisite - what does this sound like?

> - You have the command-line JSON processor (`jq`) installed.

Reply:
Hmm, I doubt this command works. I'm sorry if I was the one who suggested it. First of all it will list only pods in the current namespace, so we need to at least add -A to the command to list pods in all namespaces. 
But I'm afraid even if we delete all pods there could still be Deployments, DeploymentConfigs, Stateful sets and everything else that has a spec.runtimeClassName field. I'll try to think of something better.

Reply:
@jensfr - What is the correct command to use here?